claimManagement/src/main/java/org/openimis/imisclaims/EnquireActivity.java

```diff
@@ -186,7 +195,8 @@ public void onClick(View v) {
             public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
                 if (actionId == EditorInfo.IME_ACTION_GO) {
                     ClearForm();
-                    if (!CheckCHFID()) return false;
+                    Escape escape = new Escape();
+                    if (!escape.CheckCHFID()) return false;
```

Comment:
call escape.CheckCHFID(etCHFID.getText()) OR escape.CheckCHFID(tvCHFID.getText()) (don't know which one is here)